numba/tests/test_array_manipulation.py

```diff
@@ -516,6 +520,99 @@ def test_as_strided(self):
         self.check_as_strided(as_strided1)
         self.check_as_strided(as_strided2)
 
+    @unittest.skipIf(numpy_version < (1, 20), "requires NumPy 1.20 or newer")
+    def test_sliding_window_view(self):
+        cfunc = jit(nopython=True)(sliding_window_view)
```

Comment:
Could maybe use new `njit` decorator upon the original function itself and using thhe `.py_func` to get the python version of the function ?